src/org/rascalmpl/parser/ParserGenerator.java

```diff
@@ -63,22 +63,16 @@ public ParserGenerator(IRascalMonitor monitor, OutputStream out, List<ClassLoade
 		this.bridge = new JavaBridge(loaders, factory, config);
 		this.vf = factory;
 		
-		monitor.jobStart("Loading parser generator", 100, 139);
```

Comment:
This was one of the causes of duplicate reporting. Since these modules are "loading" anyway, but in the nested evaluator, progress bars would start duplicating information. The fact that these modules are loading, is what the user needs to see, but not twice.